kubernetes/test-infra/eks/node-config/iam_policy.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
This file is just copied directly from the AWS docs. https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.1.0/docs/install/iam_policy.json